<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="bittech-PC" name="getExcel" tests="1" failures="1" timestamp="27 Apr 2016 01:47:12 GMT" time="0.000" errors="0">
  <testcase name="VerifyInvalidLogin" time="0.000" classname="getExcel">
    <failure type="org.testng.TestNGException" message="
Some DataProvider public static java.lang.Object[][] getExcel.returnExcelSheetData(java.lang.String,java.lang.String) throws java.io.IOException parameters unresolved:  at 1 type class java.lang.String
 at 2 type class java.lang.String
">
      <![CDATA[org.testng.TestNGException: 
Some DataProvider public static java.lang.Object[][] getExcel.returnExcelSheetData(java.lang.String,java.lang.String) throws java.io.IOException parameters unresolved:  at 1 type class java.lang.String
 at 2 type class java.lang.String

	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:128)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- VerifyInvalidLogin -->
</testsuite> <!-- getExcel -->
